core/src/main/java/pl/agh/edu/actor/utils/SkinColor.java

```diff
@@ -0,0 +1,50 @@
+package pl.agh.edu.actor.utils;
```

Comment:
bardziej o czymś takim myślałem bo takie coś jest strasznie wypierdalalne np case colorCategory Transparent, trzeba podać z jakimś ColorLevelem chociaż nie ma to znaczenia. Bardziej myślałem o czym takim żeby było łatwo używalne z statycznymi importami
```java
public enum SkinColor {
	PRIMARY("primary"),
	SECONDARY("secondary"),
	ALERT("alert"),
	SUCCESS("success"),
	GRAY("gray"),
	WARNING("warning");
	final String name;
	SkinColor(String name) {
		this.name = name;
	}

	public Color getColor(ColorLevel level) {
		return GameSkin.getInstance().getColor(this.getName(level));
	}
	public String getName(ColorLevel level) {
		return name + "-" + level.value;
	}

	public enum ColorLevel {
		_000("000"),
		_100("100"),
		_300("300"),
		_500("500"),
		_700("700"),
		_900("900");

		public final String value;

		ColorLevel(String value) {
			this.value = value;
		}
	}
}

public enum SkinSpecialColor {
	TRANSPARENT("transparent"),
	SHADOW("shadow");
	final String name;

	SkinSpecialColor(String name) {
		this.name = name;
	}

	public Color getColor() {
		return GameSkin.getInstance().getColor(this.getName());
	}

	public String getName() {
		return name;
	}
}
```

Ofc można każdy enum walnąć do osobnej klasy